modules/hub/src/domain/WithPayment.ts

```diff
@@ -1,5 +0,0 @@
-import Payment from 'machinomy/dist/lib/payment'
-
-export interface WithPayment {
-  payment: Payment
-}
```

Comment:
I removed the machinomy import while trying to find a broken package. This module isn't used anywhere so I never put it back after I realized the bug was elsewhere..

Reply:
Nice, I had made a note to remove it somewhere anyway :smile: